app/js/lib/foxbox/foxbox.js

```diff
@@ -67,6 +67,8 @@ export default class Foxbox extends Service {
     this[p.settings] = new Settings();
     this[p.db] = new Db();
     this[p.net] = new Network(this[p.settings]);
+
+    this.boxes = [];
```

Comment:
I totally agree, it also clearly indicates what properties are defined for the particular class/instance. I've seen that there is a way to do this for react classes/components that likely can be linted with react eslint plugin. Maybe we can somehow enforce this for ordinary classes as well - need to dig.